app/components/inspector/associated-token/CreateIdempotentDetailsCard.tsx

```diff
@@ -0,0 +1,84 @@
+import { InspectorInstructionCard } from '@components/common/InspectorInstructionCard';
+import { MessageCompiledInstruction, ParsedInstruction, SignatureResult, VersionedMessage } from '@solana/web3.js';
+import React from 'react';
+
+export function CreateIdempotentDetailsCard(props: {
+    childIndex?: number;
+    children?: React.ReactNode;
+    index: number;
+    innerCards?: JSX.Element[];
+    ix: ParsedInstruction;
+    message: VersionedMessage;
+    raw: MessageCompiledInstruction;
+    result: SignatureResult;
+    InstructionCardComponent?: React.FC<Parameters<typeof InspectorInstructionCard>[0]>;
+}) {
+    const {
+        ix,
+        index,
+        raw,
+        message,
+        result,
+        innerCards,
+        childIndex,
+        InstructionCardComponent = InspectorInstructionCard,
+    } = props;
+
+    return (
+        <InstructionCardComponent
+            ix={ix}
+            index={index}
+            message={message}
+            raw={raw}
+            result={result}
+            title="Associated Token Program: Create Idempotent"
+            innerCards={innerCards}
+            childIndex={childIndex}
+        >
+            <tr>
+                <td>Program</td>
+                <td className="text-lg-end">
+                    <Address pubkey={ix.programId} alignRight link />
+                </td>
+            </tr>
+            <tr>
+                <td>Source</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[0]} message={message} hideInfo />
+                </td>
+            </tr>
+            <tr>
+                <td>Account</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[1]} message={message} hideInfo />
+                </td>
+            </tr>
+            <tr>
+                <td>Wallet</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[2]} message={message} hideInfo />
+                </td>
+            </tr>
+            <tr>
+                <td>Mint</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[3]} message={message} hideInfo />
+                </td>
+            </tr>
+            <tr>
+                <td>System Program</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[4]} message={message} hideInfo />
+                </td>
+            </tr>
+            <tr>
+                <td>Token Program</td>
+                <td className="text-lg-end">
+                    <AddressTableLookupAddress accountIndex={raw.accountKeyIndexes[5]} message={message} hideInfo />
+                </td>
```

Comment:
This should be included in the `CreateDetailsCard.tsx` too

Reply:
Fixed